packages/css/src/components/select/select.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@mixin reset {
+  appearance: none;
+  border: none;
+}
+
+.ams-select {
+  background-color: var(--ams-select-background-color);
+  box-shadow: var(--ams-select-box-shadow);
+  color: var(--ams-select-color);
+  font-family: var(--ams-select-font-family);
+  font-size: var(--ams-select-font-size);
+  font-weight: var(--ams-select-font-weight);
+  line-height: var(--ams-select-line-height);
+  max-inline-size: var(--ams-select-max-inline-size);
+  outline-offset: var(--ams-select-outline-offset);
+  padding-block: var(--ams-select-padding-block);
+  padding-inline: var(--ams-select-padding-inline);
+  touch-action: manipulation;
+  width: 100%;
+
+  &:not([multiple]) {
+    background-image: var(--ams-select-background-image);
+    background-position: right 1.5rem center;
+    background-repeat: no-repeat;
+    background-size: 1em 1em;
+  }
+
+  &:hover {
+    box-shadow: var(--ams-select-hover-box-shadow);
+  }
+
+  @include reset;
+}
+
+.ams-select:disabled,
+.ams-select--disabled {
+  box-shadow: var(--ams-select-disabled-box-shadow);
+  color: var(--ams-select-disabled-color);
+  cursor: not-allowed;
+
+  &:not([multiple]) {
+    background-image: var(--ams-select-disabled-background-image);
+  }
+}
+
+.ams-select--invalid,
+.ams-select:required:invalid {
```

Comment:
Do we need the `:required` part? If the select is required and doesn't have a value, it is invalid with or without that part, right?

Reply:
Copied from NLDS, I assumed it was to prevent showing red borders on a select that still needs you to select an option. Using `invalid` on input and select gives mixed results in my experience, and do not bahve the same.

Reply:
>  I assumed it was to prevent showing red borders on a select that still needs you to select an option

I don't really understand what you mean here. Is this something you've tested and is an issue in our case? If not, I wouldn't add it